.stylelintrc.cjs

```diff
@@ -47,6 +47,7 @@ module.exports = {
     'declaration-property-value-no-unknown': null,
     'function-no-unknown': [true, { ignoreFunctions: ['unit'] }],
     'media-feature-range-notation': ['prefix'],
+    'media-query-no-invalid': null,
```

Comment:
Why's it necessary to turn off this rule? There's a notes section further up (https://github.com/cfpb/consumerfinance.gov/blob/73c0aa05ebe1f390e8fc6422688b4c5227688c14/.stylelintrc.cjs#L1-L36) that talks about each rule with TODOs on turning them back on. This should get added there.